core/kernel/huk_subkey.c

```diff
@@ -105,3 +105,12 @@ TEE_Result huk_subkey_derive(enum huk_subkey_usage usage,
 	crypto_mac_free_ctx(ctx);
 	return res;
 }
+
+__weak TEE_Result huk_subkey_derive(enum huk_subkey_usage usage,
+				    const void *const_data,
+				    size_t const_data_len,
+				    uint8_t *subkey, size_t subkey_len)
```

Comment:
If we need to do this at all it should probably be `__weak __alias("__huk_subkey_derive")` instead like we do with for instance `crypto_acipher_rsaes_encrypt()`.

Reply:
Please make this:
``` diff
+TEE_Result huk_subkey_derive(enum huk_subkey_usage usage,
+                            const void *const_data, size_t const_data_len,
+                            uint8_t *subkey, size_t subkey_len)
+__weak __alias("__huk_subkey_derive");
```

Reply:
Done in https://github.com/OP-TEE/optee_os/pull/6168/commits/0dfcd0a834b0217f4643c7a297db38133c8ed35f